README.md

```diff
@@ -67,6 +67,41 @@ are available in your system, you can just add the following modules:
     module add codechecker/6.23
 
 
+Getting Started
+---------------
+
+Supported environments:
+- RHEL 9
+- Ubuntu 24.04 LTS
+
+Install dependencies:
+
+RHEL 9:
+```bash
+dnf update -y && dnf install -y llvm-toolset clang-tools-extra git python3 python3-pip gcc g++
+```
+
+Ubuntu:
+```bash
+sudo apt-get update --quiet && sudo apt-get install --no-install-recommends git python3 python3-pip python3-venv gcc g++ clang clang-tools clang-tidy
+```
+
+On ubuntu you have to make sure that `clang`, `clang-tidy` and `clang-ext-defmapping` commands are available! If they have version number appended to them use update-alternatives to get rid of the version number!
```

Comment:
You use a little too many exclamation points. This is the way I like to write as well, put some passion in an otherwise dry communication medium, but this is not the place.

Try to keep your documentation's tone a bit more formal. Reserve exclamation points for truly exceptional places (warnings, new feature announcements), that way they also carry more meaning.

Reply:
How about this:
"Some distibutions of clang, clang-tidy and clang-extdef-mapping may be installed with a trailing version number (e.g. clang-18). In case your package didn't install a non-versioned symlink as well, you will need to manually change it:"